PyInstaller/depend/utils.py

```diff
@@ -160,7 +160,14 @@ def __recursively_scan_code_objects_for_ctypes(code: CodeType):
                 # 'libgs.so.9').
                 libname = args[0]
                 if libname:
-                    libname = ctypes.util.find_library(libname)
+                    try:  # this try was inserted due to the ctypes bug https://github.com/python/cpython/issues/93094
+                        libname = ctypes.util.find_library(libname)
+                    except FileNotFoundError:
+                        libname = None
+                        logger.warn(
+                            'ctypes.util.find_library raised a FileNotFoundError. '
+                            f'Supressing and assuming no lib with the name "{args[0]}" was found.'
```

Comment:
The `logging` framework is generally optimized for % string formatting (so it can do deferred formatting). Not that it makes much difference in this case, but the rest of the codebase uses % format instead of f-strings when dealing with `logging`.

Reply:
Thanks. I changed it.